### PR TITLE
Add propagating of secret for modh-osde2e test

### PR DESCRIPTION
**Pull request by crobby**

### CODE REVIEW

File: core-services/secret-mirroring/_mapping.yaml [hunk 1]
```diff
@@ -464,3 +464,9 @@ secrets:
   to:
     namespace: ci
     name: cluster-secrets-packet
+- from:
+    namespace: managed-odh
+    name: modh-osde2e-secret
+  to:
+    namespace: ci
```
Comment (alvaroaleman) [inline core-services/secret-mirroring/_mapping.yaml:471]: Do you want to use this within test steps? If so, the namespace needs adjustments: https://docs.ci.openshift.org/docs/how-tos/adding-a-new-secret-to-ci/#use-a-secret-in-a-job-step

Comment (crobby) [reply core-services/secret-mirroring/_mapping.yaml:471]: I'm currently mirroring the approach used for other osde2e tests used for addons.  They seem to use the ci namespace.

**Review by alvaroaleman — APPROVED** [review]

**Pull request merged**